connexion/cli.py

```diff
@@ -108,6 +133,23 @@ def run(spec_file,
 
     - BASE_MODULE_PATH (optional): filesystem path where the API endpoints handlers are going to be imported from.
     """
+    if wsgi_server and server:
+        raise click.BadParameter(
+            "these options are mutually excludent",
```

Comment:
Minor typo, I guess "mutually exclusive" is the proper one =)

Reply:
I will fix my typo :)